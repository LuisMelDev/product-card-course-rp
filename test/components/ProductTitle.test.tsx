import React from 'react';
import renderer from 'react-test-renderer';
import { ProductTitle } from '../../src/components';

describe('it', () => {
  it('should render correcly', () => {
    const wrapper = renderer.create(<ProductTitle title="Hola mundo" />);
    expect(wrapper).toMatchSnapshot();
  });
});
