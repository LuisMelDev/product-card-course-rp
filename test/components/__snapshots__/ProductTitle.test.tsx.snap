// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render correcly 1`] = `
<span
  className="productDescription undefined"
>
  Hola mundo
</span>
`;
